lib/cli/package-lock.json

```diff
@@ -9,7 +9,7 @@
       "version": "7.0.0-alpha.5",
       "license": "Apache-2.0",
       "dependencies": {
-        "@alfresco/js-api": ">=7.8.0",
```

Comment:
Let's rollback the change for the cli, here js-api should have older version as we had some issues with that in the past

Reply:
So should I revert the whole file to the original version? or just this mentioned version?

Reply:
Whole file please